cmd/skywire-visor/commands/root.go

```diff
@@ -249,3 +256,27 @@ func initConfig(mLog *logging.MasterLogger, args []string, confPath string) *vis
 
 	return conf
 }
+
+func runBrowser(conf *visorconfig.V1, log *logging.MasterLogger) {
+	if conf.Hypervisor == nil {
+		log.Errorln("Cannot start browser with a regular visor")
+		return
+	}
+	addr := conf.Hypervisor.HTTPAddr
+	if addr[0] == ':' {
+		addr = "localhost" + addr
+	}
+	if addr[:4] != "http" {
+		if conf.Hypervisor.EnableTLS {
+			addr = "https://" + addr
+		} else {
+			addr = "http://" + addr
+		}
+	}
+	go func() {
+		time.Sleep(1 * time.Second)
```

Comment:
This timeout might be sufficient in most cases, but we can make it more reliable if we send a request to `/api/health`, it should respond when hypervisor is ready. What do you think of it?

Reply:
Thanks, I had an uneasy feeling when I was adding this timeout

Reply:
Just testing the webhook.

Reply:
Testing again.

Reply:
And again.

Reply:
And again

Reply:
And again